snmp/manifest.json

```diff
@@ -17,7 +18,7 @@
     "network"
   ],
   "type": "check",
-  "doc_link": "https://docs.datadoghq.com/integrations/snmp/",
+  "metric_prefix": "snmp.", // no metric in metadata?
```

Comment:
It's not clear if it should be `snmp` or `system` from the source code. metadata file is empty.
But we have an integration in the app with snmp prefix so keeping it as is